dockerfiles/Dockerfile.cross

```diff
@@ -1,5 +1,5 @@
 
-FROM    golang:1.8.1
+FROM    golang:1.8.3
```

Comment:
I'm not sure; we've had cases where a patch release brought a specific fix, using a rolling update would make it undeterministic